src/styles/card.css

```diff
@@ -0,0 +1,97 @@
+@import "./reset.css";
+@import "./stars.css";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  background: #e5e5e5;
+  color: #060b35;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+}
+
+.card__info {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 16px 0;
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
```

Comment:
didn't fixed

Reply:
You mean didn't fixed color ?

Reply:
????

Reply:
Maybe something going wrong, because I fixed in V5 margins